safe_control_gym/envs/benchmark_env.py

```diff
@@ -487,10 +489,10 @@ def after_step(self, obs, rew, done, info):
             if self.constraints is not None and self.use_constraint_penalty and self.constraints.is_violated(self, c_value=c_value):
                 if self.rew_exponential:
                     rew = np.log(rew)
-                    rew += self.constraint_penalty
```

Comment:
Is this sign change minor? Why are we changing it now?

Reply:
I switched the sign for this because i found it unintuitive that the _penalty_ is negative. The penalty should be positive, but subtracted from the reward. Pretty minor, don't think most people use this feature